rest-api-spec/src/main/resources/keywords.json

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
I don't think this file should be in `src/main/resources` as we don't want to include it in the packaged JAR. Maybe just throw it in the project root directory ?